src/lib.rs

```diff
@@ -1,7 +1,7 @@
 //! Rusty [`webpack-sources`](https://github.com/webpack/webpack-sources) port.
 
 #![feature(let_chains)]
-#![forbid(unsafe_code)]
+#![warn(unsafe_code)]
```

Comment:
Should we allow the unsafe inside the function instead of turning it off globally?